packages/core/src/Table/ColumnResizerRow.tsx

```diff
@@ -97,6 +97,47 @@ const ResizeMarker = styled.div<{
 `
 
 /**
+ * expensiveWidthLookup
+ * 
+ * This lookup gets all elements and calculates their widths to get the exact minimum size
+ */
+const expensiveWidthLookup = (
+  columnDocumentIds: React.RefObject<HTMLDivElement>,
+  clickedColumn: number,
+  currentSizes: readonly number[]
+): Array<number> => {
+
+  let maxColumnWidths: Array<number> = [...currentSizes];
+
+  for (let i = clickedColumn; i < clickedColumn + 2; ++i) {
```

Comment:
Loop seems overkill for just `clickedColumn` and `clickedColumn + 1`, better to just do this twice. Is it possible this goes out-of-bounds?